qa/setup_packages.py

```diff
@@ -81,15 +85,48 @@ def clamp_index(self, idx, cuda_version=None):
             idx = 0
         return idx
 
-    def get_name(self, cuda_version=None):
+    @staticmethod
+    def get_alias(version):
+        """Obtains alias for given version if exists. Otherwise return None
+
+            Parameters
+            ----------
+            `version`: str or PckgVer
+                Package version
+        """
+        return getattr(version, "alias", None)
```

Comment:
Just noticed, I think you always set the self.alias, so this getattr shouldn't be necessary (it will be None if you didn't provide other value).

Reply:
It is, because either you can get `PckgVer` (with alias) or just string. Or do you think all versions should be wrapped into `PckgVer`?

Reply:
Oh, this makes more sense. I think strings are fine.